designsafe/settings/common_settings.py

```diff
@@ -546,7 +546,6 @@
 
 TAPIS_SYSTEMS_TO_CONFIGURE = [
     {"system_id": AGAVE_STORAGE_SYSTEM, "path": "{username}", "create_path": True},
-    {"system_id": AGAVE_WORKING_SYSTEM, "path": "{username}", "create_path": True},
```

Comment:
can AGAVE_WORKING_SYSTEM now be removed from common_settings.py and test_settings.py?  and is now any systems deprecated and available to be removed from our WMA-Tapis-Templates (like [designsafe.storage.work.json](https://github.com/TACC/WMA-Tapis-Templates/blob/main/systems/designsafe/designsafe.storage.work.json)?)

Reply:
Good catch, removed the stragglers. And yeah I think we can deprecate that system now and remove from the repo